### PR TITLE
Also mention -- flag and ignore if undocumented

### PR DESCRIPTION
**Pull request by richsalz**
Not worth backporting.

### CODE REVIEW

File: apps/list.c [hunk 1]
```diff
@@ -337,6 +337,8 @@ static void list_options_for_command(const char *command)
             continue;
         BIO_printf(bio_out, "%s %c\n", o->name, o->valtype);
     }
+    /* Always output the -- marker since it is sometimes documented. */
+    BIO_printf(bio_out, "- -\n");
```
Comment (levitte) [inline apps/list.c]: Why the space between the dashes?

Comment (richsalz) [reply apps/list.c]: The first dash is the option, the second is what it takes (nothing).  I agree it is not obvious.

Comment (levitte) [reply apps/list.c]: Oh, ah!

**Comment by richsalz** [comment]
It would help if this can be merged soon; after cleaning up this and other nits I want to start documenting all the missing flags.

**Comment by levitte** [comment]
Done

833f7c8c55c3c8a572db2112a5c2eac92c8262b8 Also mention -- flag and ignore if undocumented

**Pull request closed without merging**